Makefile

```diff
@@ -430,10 +432,13 @@ operator-bundle: create-ocp-manifests
 	rm -rf $(BUNDLE_OUTPUT_DIR)/temp2
 	operator-sdk bundle validate $(BUNDLE_OUTPUT_DIR)
 
-# Build the bundle image.
+# Build the bundle and index images.
 .PHONY: operator-bundle-build operator-bundle-update
 operator-bundle-build:
-	podman build $(CONTAINER_BUILD_PARAMS) -f Dockerfile.bundle -t $(BUNDLE_IMAGE) .
+	docker build $(CONTAINER_BUILD_PARAMS) -f Dockerfile.bundle -t $(BUNDLE_IMAGE) .
 
 operator-bundle-update:
-	podman push $(BUNDLE_IMAGE)
+	docker push $(BUNDLE_IMAGE)
+
+operator-index-build:
+	opm index add --bundles $(BUNDLE_IMAGE) --tag $(INDEX_IMAGE) --container-tool docker
```

Comment:
Why switch to docker for this?

Reply:
Yuval preferred we stay on the same tooling until we switch everything over to podman.